op_builder/layernorm.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from .builder import Builder
+from .utils import append_nvcc_threads, get_cuda_cc_flag
+
+
+class FusedOptimBuilder(Builder):
```

Comment:
Why its name is FusedOptim?
Does it build layer_norm?

Reply:
Fixed the typo.